client/src/css/ClassForm.scss

```diff
@@ -0,0 +1,6 @@
+.class-form {
+  .popup-form {
+    background: black !important;
+    size: 'tiny';
+  }
+}
```

Comment:
You don't need the outer `class-form` rule because it's not currently being used anywhere. You can take that out and rename `popup-form` to `add-class-form` for a more descriptive name since there are multiple popup-forms in this website